README.md

```diff
@@ -136,6 +136,13 @@ makepkg -si
 $ lazysql
 ```
 
+To launch lazysql with the ability to pick from the saved connections.
+```bash
+$ lazysql [connection_url]
+```
```

Comment:
Use


````suggestion
```console
$ lazysql [connection_url]
```
````

bash or shell if for highlighting code that is shell code, not command launched in the console

Reply:
Thank you, that clears up a lot!